src/Aspire.Hosting.Azure.PostgreSQL/AzurePostgresExtensions.cs

```diff
@@ -57,7 +59,7 @@ public static class AzurePostgresExtensions
             postgres.AssignProperty(x => x.Sku.Tier, "'Burstable'");
             postgres.AssignProperty(x => x.Version, "'16'");
             postgres.AssignProperty(x => x.HighAvailability.Mode, "'Disabled'");
-            postgres.AssignProperty(x => x.Storage.StorageSizeInGB, "32");
```

Comment:
Intentional?

Reply:
No ... side effect of trying 0.1.0-beta.1 of Azure.Provisioning. Will change that back.